### DISCUSSION TITLE
Cannot add code that starts with % to user buttons

### DISCUSSION THREAD

**Discussion by jsiddall** [vlachoudis/bCNC#1676] [Q&A] [answered]
Before I even remembered that discussions existed I opened an issue on this:

https://github.com/vlachoudis/bCNC/issues/1674

However, I do think this is some kind of software issue as the same code worked for me on a different, probably older install (likely using pip install bCNC, but I don't recall anymore).

**Comment by GitHubCaps** [comment, marked as answer]
@jsiddall The issue is with changes in Python3's configparser. A few files need to be patched to get the buttons to work for both. See https://github.com/vlachoudis/bCNC/issues/1674#issuecomment-1046002120
